.github/release-drafter.yml

```diff
@@ -0,0 +1,26 @@
+categories:
+  - title: 'Major Changes'
+    labels:
+      - 'major'  # c6476b
+  - title: 'Minor Changes'
+    labels:
+      - 'feature'  # 006b75
+      - 'enhancement'  # ededed
+      - 'performance'  # 555555
+      - 'docs'  # 4071a5
```

Comment:
Is `docs` really something that warrants a release?

Reply:
That defines only the section where the docs changes go into. And yes, most docs changes deserve to be mentioned in release notes unless they have the skip tag. Sometimes they are quite important for users.

Keep in mind that this file does only helps the release-drafter decide where to mention a change, nothing else. A human can always edit the entry after and remove, sort and edit it. I usually do the manual tuning at the end to list most important changes first.